### DISCUSSION TITLE
being able to choose an external storage or a game library in another location

### DISCUSSION THREAD

**Discussion by mateowoetam** [redromnon/HeroicBashLauncher#109] [Ideas]
I recently tried to use the script for the first time, and I was hit with a message telling me to try to install Heroic or Games; I already have both, but in other locations on other drives, is there a way to select drive when running the script?

**Comment by redromnon** [comment]
Unfortunately, there's no way to select another location as of now. HBL only detects on the same drive.

Will try to work on this if possible.
